public/static/locales/en/common.json

```diff
@@ -12,6 +12,8 @@
     "copy": "Copy",
     "close": "Close",
     "closeMenu": "Close Menu",
+    "confirmDiscardChangesDialogHeader": "Discard Changes?",
+    "confirmDiscardChangesDialogMsg": "You have unsaved changes. Do you wish to continue and discard these changes?",
```

Comment:
`Do you wish to continue` seems a little confusing to me. If I were in a hurry I would anticipate this was about to say `Do you wish to continue editing/saving your changes`.  Maybe `You have unsaved changes.  Leaving this page will discard your changes.` or 411 might have something better/shorter.

Reply:
I shortened it further just to `"You have unsaved changes.` cos the title of the dialog is already says `Discard Changes?` and action button is `Discard`. 
![image](https://user-images.githubusercontent.com/1250790/121050871-f252f480-c786-11eb-8960-fd5b78d6453b.png)